source/cpu/cpu.go

```diff
@@ -173,8 +173,13 @@ func (s *cpuSource) GetLabels() (source.FeatureLabels, error) {
 	}
 
 	// Security
+	skipLabel := map[string]struct{}{
```

Comment:
you can use `k8s.io/apimachinery/pkg/util/sets` and then just:
```
sets.NewString("tdx.total_keys")
```
and check if given feature is in skipLabel by:
```
if !skipLabel.Has(k){
...
}
```

Reply:
@PiotrProkop nice suggestion, I now changed the code like that